commands/rm.go

```diff
@@ -46,50 +45,58 @@ func runRm(dockerCli command.Cli, in rmOptions) error {
 		return rmAllInactive(ctx, txn, dockerCli, in)
 	}
 
-	b, err := builder.New(dockerCli,
-		builder.WithName(in.builder),
-		builder.WithStore(txn),
-		builder.WithSkippedValidation(),
-	)
-	if err != nil {
-		return err
-	}
+	eg, _ := errgroup.WithContext(ctx)
+	for _, name := range in.builder {
+		func(name string) {
+			eg.Go(func() error {
```

Comment:
If one of the `rm` fails, shouldn't we continue to execute all of them, instead of exiting at the earliest opportunity? I think that's how the docker cli works as well.

Reply:
Yes indeed good call out: https://github.com/docker/cli/blob/c1455b6751586ee994c497013b7d49b6e4ced520/cli/command/container/rm.go#L67-L80